### PR TITLE
added celo and bsc networks

### PR DESCRIPTION
**Pull request by eliashezron**
Added celo and bsc networks to the hardhat and react app folders to enable celo and bsc developers get started with scaffold-eth

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -1,34 +0,0 @@
-name: Lint
```
Comment (codenamejason) [inline .github/workflows/lint.yaml:1]: why does this PR delete this?

Comment (eliashezron) [reply .github/workflows/lint.yaml:1]: I used gitpod ide. Must have been the environment that created that.

Comment (codenamejason) [reply .github/workflows/lint.yaml:1]: can you remove it?

**Comment by eliashezron** [comment]
unfortunately I can't. You can selectively approve file changes excluding that.

**Pull request closed without merging**